### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yingshaoxo**
add docker-compose example config

Signed-off-by: yingshaoxo <yingshaoxo@gmail.com>

### CODE REVIEW

**Comment by romange** [comment]
Thanks! Have you seen my comment about '--dir' flag?

File: README.md [hunk 1]
```diff
@@ -95,6 +95,23 @@ redis-cli PING  # redis-cli can be installed with "apt install -y redis-tools"
...
+### With docker-compose:
+```yaml
+version: "3.9"
+
+services:
+  dragonfly: # the new redis
+    image: docker.dragonflydb.io/dragonflydb/dragonfly
+    command:
+      - "--save_schedule=*:*" # save for every 1 minute
+      - "--dbfilename=/data/dump.rdb"
```
Comment (romange) [inline README.md:107]: add `--dir=/data` argument

Comment (yingshaoxo) [reply README.md:108]: updated

Comment (romange) [reply README.md:108]: You do not need a full path when you pass `--dir` :)

Comment (yingshaoxo) [reply README.md]: Modified.

**Review by romange — APPROVED** [review]

**Pull request merged**